src/main/java/tech/jhipster/lite/generator/client/svelte/core/domain/SvelteModuleFactory.java

```diff
@@ -17,96 +18,108 @@ public class SvelteModuleFactory {
-        .addScript(scriptKey("test:coverage"), scriptCommand("vitest run --coverage"))
-        .addScript(scriptKey("test:watch"), scriptCommand("vitest --"))
+    		.addScript(scriptKey("check"), scriptCommand("svelte-kit sync && svelte-check --tsconfig ./jsconfig.json"))
+        .addScript(scriptKey("lint"), scriptCommand("prettier --check . && eslint ."))
+        .addScript(scriptKey("format"), scriptCommand("prettier --write '{,src/**/,cypress/**/}*.{md,json,js,cjs,svelte,css,html,yml}'"))
+        .addScript(scriptKey("test"), scriptCommand("vitest run --coverage"))
+        .addScript(scriptKey("test:ui"), scriptCommand("vitest --ui --coverage"))
+        .addScript(scriptKey("test:watch"), scriptCommand("vitest"))
         .and()
       .optionalReplacements()
         .in(path("package.json"))
           .add(lineBeforeText(ENGINES_NEEDLE), type(properties.indentation()))
           .and()
         .and()
       .files()
-        .add(SOURCE.file(".eslintrc.cjs"), to(".eslintrc.cjs"))
-        .add(SOURCE.file("tsconfig.json"), to("tsconfig.json"))
-        .add(SOURCE.file("vite.config.js"), to("vite.config.js"))
         .add(SOURCE.append("src/main/webapp/routes").template("+page.svelte"), to("src/main/webapp/routes/+page.svelte"))
         .add(PRIMARY_MAIN_SOURCE.template("App.svelte"), PRIMARY_MAIN_DESTINATION.append("App.svelte"))
-        .add(PRIMARY_TEST_SOURCE.template("App.spec.ts"), PRIMARY_TEST_DESTINATION.append("App.spec.ts"))
+        .add(PRIMARY_MAIN_SOURCE.template("App.spec.js"), PRIMARY_MAIN_DESTINATION.append("App.spec.js"))
         .batch(CLIENT_COMMON, to("."))
-          .addFile(".eslintignore")
           .addFile(".npmrc")
           .and()
         .batch(SOURCE, to("."))
           .addTemplate("svelte.config.js")
-          .addTemplate("vitest.config.ts")
+          .addTemplate("eslint.config.js")
+          .addTemplate("vite.config.js")
+          .addFile("vitest-setup.js")
+          .addTemplate("jsconfig.json")
           .and()
         .batch(SOURCE.file("src/main/webapp"), to("src/main/webapp"))
           .addTemplate("app.html")
-          .addTemplate("app.d.ts")
           .and()
         .batch(SOURCE.file("src/main/webapp/assets"), to("src/main/webapp/assets"))
-          .addFile("JHipster-Lite-neon-orange.png")
+          .addFile("favicon.png")
+          .and()
+        .batch(SOURCE.file("src/main/webapp/assets/img"), to("src/main/webapp/assets/img"))
+          .addFile("jhipster-lite-neon-orange.png")
           .addFile("svelte-logo.png")
           .and()
         .and()
+          .apply(patchPrettierConfig())
       .build();
     //@formatter:on
   }
 
+  private Consumer<JHipsterModuleBuilder> patchPrettierConfig() {
+    //@formatter:off
+    return moduleBuilder -> moduleBuilder
+      .mandatoryReplacements()
+        .in(path(".prettierrc"))
+          .add(text("plugins:"), "plugins:\n  - prettier-plugin-svelte")
```

Comment:
You have a more convenient way of doing that:
```suggestion
          .line(lineBeforeText("plugins:"), "- prettier-plugin-svelte")
```
Note that in order to respect configured indentation, you should even do:
```suggestion
          .line(lineBeforeText("plugins:"), properties.idendation().times(1) + "- prettier-plugin-svelte")
```